fetch_data/utils/get_wikibase.py

```diff
@@ -3,44 +3,66 @@
 from sqlalchemy import select
 from sqlalchemy.orm import joinedload
 from sqlalchemy.ext.asyncio import AsyncSession
+from logger import logger
 from model.database import WikibaseModel
 
 
+# pylint: disable-next=too-many-arguments,too-many-positional-arguments
 async def get_wikibase_from_database(
     async_session: AsyncSession,
     wikibase_id: int,
-    include_observations: bool = False,
-    require_action_api: bool = False,
+    join_connectivity_observations: bool = False,
+    join_log_observations: bool = False,
+    join_property_observations: bool = False,
+    join_quantity_observations: bool = False,
+    join_statistics_observations: bool = False,
+    join_user_observations: bool = False,
+    join_version_observations: bool = False,
+    require_article_path: bool = False,
+    require_script_path: bool = False,
     require_sparql_endpoint: bool = False,
-    require_special_statistics: bool = False,
-    require_special_version: bool = False,
 ) -> WikibaseModel:
     """Get Wikibase"""
 
+    logger.debug("Fetching Wikibase", extra={"wikibase": wikibase_id})
+
     query = select(WikibaseModel).where(WikibaseModel.id == wikibase_id)
-    if include_observations:
+    if join_connectivity_observations:
+        query = query.options(joinedload(WikibaseModel.connectivity_observations))
+    if join_log_observations:
+        query = query.options(joinedload(WikibaseModel.log_month_observations))
+    if join_property_observations:
         query = query.options(
-            joinedload(WikibaseModel.connectivity_observations),
-            joinedload(WikibaseModel.log_month_observations),
-            joinedload(WikibaseModel.property_popularity_observations),
-            joinedload(WikibaseModel.quantity_observations),
-            joinedload(WikibaseModel.software_version_observations),
-            joinedload(WikibaseModel.statistics_observations),
-            joinedload(WikibaseModel.user_observations),
+            joinedload(WikibaseModel.property_popularity_observations)
         )
+    if join_quantity_observations:
+        query = query.options(joinedload(WikibaseModel.quantity_observations))
+    if join_statistics_observations:
+        query = query.options(joinedload(WikibaseModel.statistics_observations))
+    if join_user_observations:
+        query = query.options(joinedload(WikibaseModel.user_observations))
+    if join_version_observations:
+        query = query.options(joinedload(WikibaseModel.software_version_observations))
```

Comment:
If we wanna be strict about the naming, the flag should be `join_software_version_observations`.